ui/partials/main/jobs.html

```diff
@@ -6,6 +6,12 @@
   <div class="progress-bar"  role="progressbar" style="width: 100%"></div>
 </div>
 
+<div uib-alert type="info" style="flex: none;" ng-show="serverChangedDelayed" ng-cloak>
+    <i class="fa fa-info-circle" aria-hidden="true"></i>
+    Treeherder has updated. To pick up the changes, you can reload the page
+    <button ng-click="updateButtonClick()" class="btn btn-xs btn-danger">Reload</button>
+  </div>
```

Comment:
Putting it here doesn't work very well as it causes the banner to be hidden if the user scrolls out of view.

I think we'd better off putting this in a new partial, and including it just before th-global-content in index.html